client/.env

```diff
@@ -1 +1,3 @@
 EXTEND_ESLINT=true
+REACT_APP_AIRTABLE_API_KEY="keyczjtMi7gM7IUKE" 
+REACT_APP_AIRTABLE_BASE="appi9N1EnrGen2B1g"
```

Comment:
Please remove these are they are not needed on the frontend anymore. Also we try to avoid commiting secrets onto the repos.

Reply:
Got it! I just removed it and pushed the updated version.